scraper.rb

```diff
@@ -7,7 +7,16 @@
      WikiData::Category.new('Category:Women members of the House of Representatives (Nigeria)', 'en').member_titles |
      WikiData::Category.new('Category:People\'s Democratic Party members of the House of Representatives (Nigeria)', 'en').member_titles |
      WikiData::Category.new('Category:Members of the House of Representatives (Nigeria) from Rivers State', 'en').member_titles |
-     WikiData::Category.new('Category:Speakers of the House of Representatives (Nigeria)', 'en').member_titles 
+     WikiData::Category.new('Category:Speakers of the House of Representatives (Nigeria)', 'en').member_titles
 
-EveryPolitician::Wikidata.scrape_wikidata(names: { en: en })
+sparq = <<EOQ
+  SELECT ?item WHERE {
+    ?item p:P39 ?position_held .
+    ?position_held ps:P39 wd:Q21290864 .
+    ?position_held pq:P2937 wd:Q20311259
```

Comment:
This is fine, but in case you were unaware, when you have two statements for the same subject, you can join them with a semicolon instead, i.e.:

```sparql
    ?position_held ps:P39 wd:Q21290864 ;
                          pq:P2937 wd:Q20311259 .
```